extensions/wikia/TemplateClassification/Permissions.class.php

```diff
@@ -15,8 +15,18 @@ class Permissions {
 	 * @return bool
 	 */
 	public function shouldDisplayEntryPoint( \User $user, \Title $title ) {
+		return $this->userCanChangeType( $user, $title )
+		       && $title->inNamespace( NS_TEMPLATE );
```

Comment:
Actually it's an inconsistency between [MediaWiki](https://www.mediawiki.org/wiki/Manual:Coding_conventions#Line_continuation) and Wikia guidelines. I personally find the MW ones better since the last line is not out of context of a logical condition if you grep for it. Actually, if you look at the guidelines, they do not specify what should happen to the operator whereas the MW ones do.

P.S. Damn, we debate on such an important stuff, don't we?